.gitignore

```diff
@@ -1,2 +1,7 @@
+# npm
+node_modules
```

Comment:
Ah! Sorry I'm not sure I 100% understand - do you mean changing this line to more specifically target dgraph/node_modules rather than just node_modules? Or do you mean removing the dgraph/graphql prefix from line 5?

Reply:
Ohhh never mind I see what you meant! I just realised this .gitignore was one level above the Dgraph folder.